org.eclipse.lsp4e/src/org/eclipse/lsp4e/DocumentContentSynchronizer.java

```diff
@@ -71,7 +72,7 @@ final class DocumentContentSynchronizer implements IDocumentListener {
 
 	private int version = 0;
 	private DidChangeTextDocumentParams changeParams;
-	private long modificationStamp;
```

Comment:
not very important, but why do we modify the name of this field? It looks like it is still the modificationTimestamp, isn't it?

Reply:
modify can mean 2 things: if I modify a document with `document.set(...)`, then it doesn't change themodificationStamp in this class - the stamp is only read on open and save. Changing the name is just for disambiguation to clarify that it's not the modification of the document content, but really the "storage" stamp.